examples/dictionary/test/dictionary_integration.ts

```diff
@@ -9,10 +9,10 @@ describe('Dictionary pack integration test', () => {
     // Here we execute the formula using a real http fetcher. Since this pack requires authentication,
     // this requires that you've already run `coda auth examples/dictionary/manifest.ts` to set up
     // an API key.
-    const response = await executeFormulaFromPackDef(manifest, 'Define', ['coda'], undefined, undefined, {
+    const response = (await executeFormulaFromPackDef(manifest, 'Define', ['coda'], undefined, undefined, {
```

Comment:
Nit, could we make is so that you invoke this as `await executeFormulaFromPackDef<any[]>(...)`? Would prefer not to have casts be required to write unittests if using TS.

I would maybe define executeFormulaFromPackDef to be something like

```
export function executeFormulaFromPackDef<T extends PackFormulaResult = any> {
```

so that we get the old, more flexible behavior by default.

Reply:
right. we just talked about this. haven't got a chance to reflect yet. let me update the sdk.

Reply:
done.